custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssueNavigator.kt

```diff
@@ -1,41 +1,66 @@
 package jces1209.vu.page
 
+import jces1209.vu.wait
 import org.openqa.selenium.By
 import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.ui.ExpectedConditions
 import org.openqa.selenium.support.ui.ExpectedConditions.and
 import org.openqa.selenium.support.ui.ExpectedConditions.or
 import org.openqa.selenium.support.ui.ExpectedConditions.presenceOfElementLocated
 import java.time.Duration
+import java.util.*
 
 class CloudIssueNavigator(
     driver: WebDriver
 ) {
+    private val webDriver = driver
     private val page = FalliblePage.Builder(
-            driver,
-            or(
-                and(
-                    or(
-                        presenceOfElementLocated(By.cssSelector("ol.issue-list")),
-                        presenceOfElementLocated(By.id("issuetable")),
-                        presenceOfElementLocated(By.id("issue-content"))
-                    ),
-                    or(
-                        presenceOfElementLocated(By.id("jira-issue-header")),
-                        presenceOfElementLocated(By.id("key-val"))
-                    ),
-                    or(
-                        presenceOfElementLocated(By.id("new-issue-body-container")),
-                        presenceOfElementLocated(By.className("issue-body-content"))
-                    )
+        driver,
+        or(
+            and(
+                or(
+                    presenceOfElementLocated(By.cssSelector("ol.issue-list")),
+                    presenceOfElementLocated(By.id("issuetable")),
+                    presenceOfElementLocated(By.id("issue-content"))
                 ),
-                presenceOfElementLocated(By.className("no-results-hint")) // TODO is it too optimistic like in SearchServerFilter.waitForIssueNavigator ?
-            )
+                or(
+                    presenceOfElementLocated(By.id("jira-issue-header")),
+                    presenceOfElementLocated(By.id("key-val"))
+                ),
+                or(
+                    presenceOfElementLocated(By.id("new-issue-body-container")),
+                    presenceOfElementLocated(By.className("issue-body-content"))
+                )
+            ),
+            presenceOfElementLocated(By.className("no-results-hint")) // TODO is it too optimistic like in SearchServerFilter.waitForIssueNavigator ?
         )
+    )
         .cloudErrors()
         .timeout(Duration.ofSeconds(30))
         .build()
 
     fun waitForNavigator() {
         page.waitForPageToLoad()
     }
+
+    fun chooseElement(): WebElement {
```

Comment:
It has the confusing name and it's redundant
you can get webelement in `switchBetweenIssuesInSearchResults` method or make it private and rename like 'getIssueElementFromList'